stages/org.osbuild.users.meta.json

```diff
@@ -65,6 +65,10 @@
               "expiredate": {
                 "description": "The date on which the user account will be disabled. This date is represented as a number of days since January 1st, 1970.",
                 "type": "integer"
+              },
+              "password_changed_date": {
```

Comment:
(super niptick) I was thinking about the name a bit, I'm not super happy with this one but `lastday` (as from the chage tool) is much less appropriate and I can't think of anything better so it seems it's a good pick :) But maybe someone else has ideas?

Reply:
My thoughts exactly 😆 I would be happy to change it to something better

Reply:
How about `expire_date` or `expire_time` ?

Reply:
I think `expire_date` and `expire_time` would be confusing because we already have `expiredate` for user expiry, and those names don't make it clear that the password is expiring, not the, user itself. Also, this date is not a date at which the password expires. It is the date at which the password was last changed. If you have a policy where a password expires 90 days after the last change, the expire date would be 90 days after the date given in this field.

Reply:
I like the idea of having the property name match the actual option but (as Michael said), `lastday` wouldn't mean anything.  An alternative would be to nest the option under a `chage` property to make it clearer:
```json
"chage": {
  "lastday": "..."
}
```
which also leaves open the option of adding other options to the object in the future, like `inactive` or `min/maxdays`.

Another alternative (but requires more work and might not be worth it) would be to make it a completely separate stage `org.osbuild.chage`, but that might be overkill.

Reply:
I am okay with the `chage` object property if no one is apposed.

Reply:
Sounds like enough people think that a full `chage` stage would be overkill that just a simple `force_reset` flag is the way to go.